### DISCUSSION TITLE
agenda for meeting on Jan 19

### DISCUSSION THREAD

**Discussion by venkateshpotluri** [make4all/psst#91] [Announcements]
@make4all/sensor-accessibility  The primary agenda for this meeting is to go over the implementation of the architecture in #90.
Some things to think about (we will re-visit these after discussing the code):

- [ ] how do we handle fast flowing data, and the sonification of each datapoint? can we draw from any visualization techniques used to visualize large amounts of fast-streaming data?
- [ ] What would a GUI to select and parameterize templates look like?
- [ ] How would this library handel real data coming from a real sensor? what would be a good first demo?
- [ ] What other demos can we build?

Please feel free to comment if you would like to add anything else to the agenda or mention at the beginning of the meeting.

**Comment by jmankoff** [comment]
Discussion:
- Consider adding the ability to play data at it's Datum timestamp in the sonifier
- Think about how to schedule stuff given timestamps
- Seems like scheduling the data is a tricky problem for sonification ... library should help with that, or take it on. 
- Who subsamples
Next steps planned:
- @jamesadevine will hook up to jacdac
- @jrthompson33 will try out templates/making a new template
- @bongshin meeting while trying the demo to show potentially 
- Continuing to think through the GUI experience is important too
- @venkateshpotluri will try to work on the noise audio node and potentially adding support for pause.

**Comment by venkateshpotluri** [comment]
@jamesadevine  I just merged the code with the architecture we discussed this morning. This should release an updated version of the package. Alternative to using the package, you could potentially create a new demo page in the same repository which pulls data from Jacdac.
